docs/topics/db/examples/many_to_many.txt

```diff
@@ -64,7 +64,9 @@ Save it!
 
     >>> a1.save()
 
-Associate the ``Article`` with a ``Publication``::
+Associate the ``Article`` with a ``Publication``,
```

Comment:
This page is meant to be quite introductory, and doesn't mention signals. Could this be added to the documentation for `django.db.models.signals.m2m_changed` instead?